iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/client/DataNodeRequestType.java

```diff
@@ -32,7 +32,6 @@ public enum DataNodeRequestType {
   STOP_REPAIR_DATA,
   LOAD_CONFIGURATION,
   SET_SYSTEM_STATUS,
-  BROADCAST_LATEST_CONFIG_NODE_GROUP,
```

Comment:
We could keep the BROADCAST_LATEST_CONFIG_NODE_GROUP interface, since it's never too much to broadcast immediately when the cluster ConfigNode group has been updated. Therefore the services that rely on ConfigNodeList will save the 1s heartbeat interval time. The heartbeat function should be took as a security method to ensure the DataNodes can update their ConfigNode list when BROADCAST_LATEST_CONFIG_NODE_GROUP failed.

Reply:
This is indeed worth considering, and we can have a discussion. I believe there is no need to retain the original RPC notification mechanism here, as the ConfigNode list is not time-sensitive. After a ConfigNode is added, there doesn't seem to be anything that requires an immediate response from the DataNode.